### PR TITLE
[READY] Resolve some open Ansible issues

### LINKED ISSUE #207: investigate isc-dhcp-server6 coredump
isc-dhcp-server6 core dumps within a couple minutes of running

### LINKED ISSUE #226: make dhcp server force faster lease expire after restart
this is a 'can we?'

rather than letting clients renew a lease 3 times, after a restart, deny renewals for one renewal time so that the changes take effect faster.

### LINKED ISSUE #232: switch from .tsv to .csv
it's much easier to run .csv through a spreadsheet to edit a column than a .tsv (it can be done, but there are extra steps)

### PR DESCRIPTION
**Pull request by kylerisse**
## Description of PR
Fixes #207
Fixes #226
Fixes #232

This PR includes fixes for 3 open issues as well as some general housekeeping within the Ansible directory.

## Previous Behavior
- isc-dhcp-server6 service would go OOM during startup or shortly after due to unnecessarily large IPv6 scope space
- IPv4 leases could be renewed for up to 15 minutes on a 5 minute lease, preventing APs from getting timely DHCP related configuration option updates
- Data files were tab separated instead of comma 
- Python2 was still being installed on servers
- Ansible file layout was unnecessarily cumbersome

## New Behavior
- Greatly reduce the IPv6 DHCP scope sizes to address issue #207 . DHCP scope is now 1024 per subnet for each DHCP server. The service no longer crashes with coredump/OOM conditions.
- Set the IPv4 DHCP max lease time equal to the default lease to address issue #226 . The server has no control over clients requesting renewals within the 5 minutes but will no longer allow them after a full first lease.
- Switched fact files from TSV to CSV to address issue #232 . This includes the lists for servers, routers, aps, and pis.
- Removed the python2 role from the project. It will be deprecated before the next SCaLE.
- Simplified the ansible file layout by removing the embedded etc/ansible directory.
- Added a header row to the AP list

## Tests
Test for Issue #207 
1. bring environment up with vagrant up
2. login to server1 with vagrant ssh server1, elevate to root
3. open file to edit /etc/netplan/50-vagrant.yml
4. add item to addresses list - 2001:470:f325:503::5/64 and save file
5. netplan apply
6. verify 2001:470:f325:503::5/64 attached to enp0s8
7. restart isc-dhcp-server6 with systemctl restart isc-dhcp-server6
8. tail dhcp log with tail -f /var/log/dhcp.log
9. in another terminal log in to server8 with vagrant ssh server8, elevate to root
10. created 200 test dhcp requests, for i in {1..200}; do dhclient -6; done;
11. in server1 terminal verify logs
12. verify isc-dhcp-server6 has not crashed with systemctl status isc-dhcp-server6 and viewing the "Active" item

Test for Issue #226 
1. login to server1 with vagrant ssh server1
2. review /etc/dhcp/dhcpd.conf and note that default-lease-time and max-lease-time now match

Test for Issue #232 
1. view the following files in the facts directory {pi/pilist.csv, aps/aplist.csv, servers/serverlist.csv, routers/routerlist.csv} and note they are all csv now
2. run inventory.py and note data structure populated as expected. ./inventory.py | jq "." | less

### CODE REVIEW

**Comment by owendelong** [comment]
I'm not sure I'm wild about converting everything to csv vs. tsv. Having said that, if we're really determined to move forward with it, then I think the Google sheet integration is a critical first step rather than the other way around.

Editing CSVs is a huge PITA and for the most part, I've been the one maintaining these files and using vi to do it.

If we're dead set on doing this, then there's a python script as part of this project (https://www.sparkfun.com/sparkx/blog/2591)  which extracts a CSV from a google sheet automatically that you might find useful.

**Comment by kylerisse** [comment]
Commented on the issue also. Just wondering, when I look at the 4 files in question i see you have committed to all of them combined a total of zero times. @davidelang and I have maintained the APs list and this is what we discussed with everyone in the NOC at SCaLE and there were no objections then. I'm not sure exactly what the protest is. I'll leave the PR here. If there was an objection to the work then it would have been nice if it was noted in the issue rather than waiting for the work and testing to deliver a PR.

**Comment by owendelong** [comment]
I thought I saw some of the network files (routers, vlans, etc.) involved or planned for this.

If it's just the stuff you guys are maintaining, knock yourselves out.

File: ansible/scale.yml [hunk 1]
```diff
@@ -3,36 +3,27 @@
...
     - dnsserver
     - dhcpserver
     - zabbixagent
 
 - hosts: servers
   become: yes
-  vars:
-    ansible_python_interpreter: /usr/bin/python3
   roles:
-    - python27
```
Comment (sarcasticadmin) [inline ansible/scale.yml]: Nice to see python27 go away 👍

File: ansible/Vagrantfile [hunk 2]
```diff
@@ -1,7 +1,7 @@
 # -*- mode: ruby -*-
 # vi: set ft=ruby :
 
-playbook = 'etc/ansible/scale.yml'
```
Comment (sarcasticadmin) [inline ansible/Vagrantfile]: Thanks for simplifying this

File: ansible/roles/dhcpserver/files/dhcpd.conf [hunk 3]
```diff
@@ -3,7 +3,7 @@
 option domain-name "scale.lan";
 
 default-lease-time 300;
-max-lease-time 900;
+max-lease-time 300;
```
Comment (sarcasticadmin) [inline ansible/roles/dhcpserver/files/dhcpd.conf]: Looks good

File: ansible/inventory.py [hunk 4]
```diff
@@ -126,9 +126,9 @@ def dhcp6ranges(prefix, bitmask):
     prefsplit = re.split(r'\:\:', prefix)[0]
     return [
         prefsplit + ":1::1",
-        prefsplit + ":0fff::fffe",
-        prefsplit + ":f000::1",
-        prefsplit + ":ffff::fffe",
+        prefsplit + ":1::400",
```
Comment (sarcasticadmin) [inline ansible/inventory.py]: Alright now ipv6 dhcp should work!

**Comment by sarcasticadmin** [comment]
@owendelong going to dismiss your request for changes since it sounds like you and @kylerisse have worked it out

**Pull request merged**